src/components/Settings/AdvancedSettings.tsx

```diff
@@ -36,6 +43,7 @@ export default function TitleChanger() {
             placeholder="/favicon.svg"
             {...form.getInputProps('favicon')}
           />
+          <TextInput label="Background" placeholder="" {...form.getInputProps('background')} />
```

Comment:
Placeholder should be added
```suggestion
          <TextInput label="Background" placeholder="/img/background.jpg" {...form.getInputProps('background')} />
```

Reply:
We don't suggest mounting any directories outside of `/public/icons`, so I wasn't really sure what to do here. I can suggest a URL or perhaps a change to the wiki/readme to mount `/public` instead of `/public/icons`.

Reply:
Then what do you suggest? It doesn't make sense to suggest people mount an icons folder and then ask them to link to a file outside of the only volume they have mounted.

Reply:
Just make it background.jpg, and revert the changes to the mounting. A placeholder is meant to give people an idea of what they should do.